src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    public static final int PRICE = 1000;
+    private Set<LottoNumber> lotto;
```

Comment:
```suggestion
    public static final int PRICE = 1000;
    
    private Set<LottoNumber> lotto;
```
상수와 변수 사이에 줄바꿈을 넣어서 가독성을 좋게하면 어떨까요?:)